File: ProvaGFT/src/exercicio01/Vendedor.java
package exercicio01;

public class Vendedor extends Funcionario {

	public Vendedor(String nome, int idade, double salario) {
		super(nome, idade, salario);
		
	}
	
	@Override
	public void Bonificacao() {
		this.setSalario(this.getSalario()+3000.00);		
		System.out.println(this.getNome() + ", com o salário de " + this.getSalario());
	}
	

}

File: Prova/Vendedor.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Prova
{
    class Vendedor : Funcionarios
    {
        public Vendedor(string nome, int idade, double salario) : base(nome, idade, salario)
        {
        }


        public void Bonificacao()
        {
            Salario = Salario + 3000.00;
            Console.WriteLine("O salário do " + Nome + " é de " + Salario+ " reais.");
        }

    }
}
